ecl/eclcc/WORKUNITS.rst

```diff
@@ -0,0 +1,862 @@
+=======================
+Understanding workunits
+=======================
+
+Introduction
+************
+
+A workunit contains all the information that the system requires about a query - including what parameters it takes,
+how to execute it, and how to format the results.  Understanding the contents of a workunit is a key step to
+understanding how the HPCC system fits together.  This document begins with an overview of the different elements
+in a workunit.  That is followed by a walk-through executing a simple query, with a more detailed description of
```

Comment:
in a workunit, followed by a 
or
workunit. This is then followed by
or
workunit. At the end (something something)
etc etc